File: LeetCode-Programs/Find Lucky Integer in an Array.cpp
class Solution {
public:
    int findLucky(vector<int>& arr) {
        vector<int> num;
        for(int i=0;i<arr.size();i++)
        {
            int count =0;
            for(int j=0;j<arr.size();j++)
            {
                if(arr[i]==arr[j])
                    count++;
            }
            if(count==arr[i])
                num.push_back(arr[i]);
        }
        sort(num.begin(),num.end());
        if(num.size()==0)
            return -1;
        return num[num.size()-1];
    }
};

File: LeetCode-Programs/Find Lucky Integer in an Array.py
class Solution:
    def findLucky(self, arr: List[int]) -> int:
        new_list=[]
        for i in arr:
            count=0
            for j in arr:
                if i==j:
                    count=count+1
            if count==i:
                new_list.append(i)
                
        new_list.sort()
        if len(new_list)==0:
            return -1
        return new_list[len(new_list)-1]
